nba_relations_analysis.R

```diff
@@ -1,9 +1,12 @@
-library(ggplot2)
 library(tidyverse)
-
+library(ggplot2)
+library(plotly)
 nba_players <- read_csv("C:\\Users\\amariik\\Documents\\learning_R\\datasets\\NBA_PLAYERS.csv")
```

Comment:
```suggestion
nba_players <- read_csv("learning_R/datasets/NBA_PLAYERS.csv")
```
Absolute path needs to be changed to relative path.